.github/actions/deploy-release-candidate-v1/README.md

```diff
@@ -0,0 +1,37 @@
+# deploy-release-candidate-v1
+
+A GitHub Action to prepare already created release candidate for QA.
+
+## Inputs
+
+| Name            | Required | Description                                                | Default |
+| --------------- | -------- | ---------------------------------------------------------- | ------- |
+| `version`       | Yes      | A version of the release candidate                         | NA      |
+| `revision-url`  | Yes      | A commit revision URL                                      | NA      |
+| `owner`         | Yes      | An owner of the repository                                 | NA      |
+| `repo`          | Yes      | A repository name                                          | NA      |
+| `slack-webhook` | Yes      | A Slack channel webhook URL where the message will be sent | NA      |
+| `slack-channel` | No       | A Slack channel name where the message will be sent        | NA      |
```

Comment:
This is probably fine to hardcode into the action. I don't think we should be posting these release candidate build messages to different slack channels so shouldn't give the user an option. We can change that in the future if that ever happens and just have this default to `api-team-releases`.